src/components/NFTPage/Recommendation.vue

```diff
@@ -153,27 +157,26 @@ export default {
         : [...DEFAULT_RECOMMENDATIONS_LIST.BOOK.map(nft => ({ classId: nft }))];
     },
     normalizedList() {
-      const currentList = [...this.recommendedList];
-      if (this.recommendedList.length < 5) {
```

Comment:
Maybe just cast `this.displayItemCount ` as a number here, and use `const displayItemCount ` in below
i.e. `const displayItemCount = Number(this.displayItemCount)`